models/filtermodel.cpp

```diff
@@ -22,6 +23,7 @@
 
 FilterModel::FilterModel(QObject *parent)
     : QSortFilterProxyModel(parent)
+    , m_sortParameter(SortParameter::None)
```

Comment:
Currently, it's the case. But in fact, tracker seems to provide document sorted by date in fact. Can we reflect this by initialising this at sorted by date ?

Reply:
Modified

Reply:
What do you mean? Sorting by date by default?

Reply:
Ok, I'll do it

Reply:
Done